ui/lib/theme.ts

```diff
@@ -97,8 +98,16 @@ export const GlobalStyle = createGlobalStyle`
 export const muiTheme = createTheme({
   typography: { fontFamily: "proxima-nova" },
   palette: {
+    primary: {
+      main: "#006B8E",
```

Comment:
Nit: comment about what this color is, like "Weave Blue" or something.

Reply:
YES